.gitconfig

```diff
@@ -0,0 +1,3 @@
+[remote "upstream"]
+        fetch = +refs/pull/*:refs/remotes/pr/*
+        fetch = +refs/merge-requests/*:refs/remotes/pr/*
```

Comment:
In 3ecd2fbccc64bc707a9f418c93e019192e9113d9:

You should not have committed this file

Reply:
it's deleted in the next commit, but could still cause weird things to happen during bisection.

Reply:
Removed